src/ucp/core/ucp_ep.h

```diff
@@ -372,10 +382,8 @@ struct ucp_ep_config {
         ucp_ep_rndv_zcopy_config_t get_zcopy;
         /* RNDV PUT Zcopy configuration */
         ucp_ep_rndv_zcopy_config_t put_zcopy;
-        /* Threshold for switching from eager to RMA based rendezvous */
-        ucp_rndv_thresh_t          rma_thresh;
-        /* Threshold for switching from eager to AM based rendezvous */
-        ucp_rndv_thresh_t          am_thresh;
+        /* Threshold for switching from eager to rendezvous */
+        ucp_eager_rndv_thresh_t    thresh;
```

Comment:
imo no need to include `eager` to the name
maybe `ucp_ep_rndv_thresh_t`?

Reply:
Reverted related changes.